projects/exercises/01/drive_in_line.cpp

```diff
@@ -0,0 +1,13 @@
+#include <STSL/RJRobot.h>
+
+using namespace std;
+
+int main()
+{
+    RJRobot robot(RobotType::REAL); //Initialize a RJ Robot that controls a real robot
```

Comment:
```suggestion
    RJRobot robot; //Initialize a RJ Robot
```

No sim support in STSL this year.